openstack_commands/owned_networks.sh

```diff
@@ -6,6 +6,4 @@ openstack subnet create \
           --network nerc-infra-routed \
           --subnet-range 10.85.0.0/22  \
           --ip-version 4 \
-          --allocation-pool start=10.85.1.0,end=10.85.3.254 \
-          --host-route destination=10.30.9.0/24,gateway=10.85.0.1 \
-          --dhcp subnet-nerc-infra-routed
+          subnet-nerc-infra-routed
```

Comment:
I think you may want to specify `--no-dhcp`; I'm not sure what the default behavior is otherwise.